apps/services/open_feature/providers/dev_cycle/feature_flags.py

```
@@ -0,0 +1,257 @@
+                    evaluation_context=context
+                )
+                
+                # Check if flag was found using the reason
+                reason_str = str(details.reason) if details.reason is not None else "DEFAULT"
+                if reason_str not in ["FLAG_NOT_FOUND", "DEFAULT"]:
+                    logger.debug(f"Provider flag {feature_key}: {details.value} (reason: {reason_str})")
+                    return details.value
+                else:
+                    logger.debug(f"Provider flag {feature_key} not found, using environment fallback")
+                    
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)